Makefile

```diff
@@ -60,6 +60,8 @@ test-integration:
 test-gaia:
 	@go test -mod=readonly -v -run TestGaiaToGaiaRelaying ./_test/
 	@go test -mod=readonly -v -run TestRelayAllChannelsOnConnection ./_test/
+	@go test -mod=readonly -v -run TestUnorderedChannelBlockHeightTimeout ./_test/
+	@go test -mod=readonly -v -run TestUnorderedChannelTimestampTimeout ./_test/
```

Comment:
Could this all be a single `go test` invocation with a run regex, like `go test -mod=readonly -v -run 'TestGaiaToGaiaRelaying|TestGaiaToGaiaRelaying|TestUnorderedChannelBlockHeightTimeout|TestUnorderedChannelTimestampTimeout' ./_test`? If we still need to force only one test running at a time, then we can use `-parallel 1' too.

Reply:
I don't think there is any reason why we would need to force one test running at a time, gonna make this adjustment rn!